ci/docker/python-free-threaded-wheel-windows-test-vs2022.dockerfile

```diff
@@ -37,16 +37,13 @@ ENV PYTHON_CMD="py -${python}t"
 SHELL ["cmd", "/S", "/C"]
 RUN %PYTHON_CMD% -m pip install -U pip setuptools
 
-COPY python/requirements-wheel-test.txt C:/arrow/python/
+COPY python/requirements-wheel-test-3.13t.txt C:/arrow/python/
```

Comment:
Python 3.14 is going to be released around the next Arrow release (2025-10-07), we might want to add wheels for Arrow 22.
Can you add a comment here to revisit for Python 3.14? We will probably have to make this conditional

Reply:
Or just open an issue for 3.14.

Reply:
I planned to open an issue for 3.14 - forgot to add in the comments!
Will do asap.

Reply:
Done: https://github.com/apache/arrow/issues/47437